scrapers/mi/bills.py

```diff
@@ -13,12 +13,13 @@
 
 def jres_id(n):
     """joint res ids go from A-Z, AA-ZZ, etc."""
+    # Offset by 1 to account for zero
+    n = n + 1
     if n <= 26:
         return chr(ord('@') + n)
-    elif n > 26:
-        return chr(ord('@') + n % 26) + chr(ord('@') + n % 26)
     else:
-        return n
+        offset = n % 26 if n % 26 == 0 else 26
+        return chr(ord('@') + offset) + chr(ord('@') + offset)
```

Comment:
Probably worth documenting the logic here a bit. May be difficult for someone to track through this later.

Reply:
Good point, I will add in some explanation here and an example